templates/_linux.yaml

```
@@ -43,7 +45,8 @@ objects:
 {% endif %}
 
           cpu:
-            cores: {{ item.cores }}
+            sockets: {{ item.sockets }}
```

Here I would define all three topology attributes this way (for now):

```suggestion
            sockets: {{ item.cpus }}
            cores: 1
            threads: 1
```

Also do not forget to change this in the windows template as well.